inspirehep/dojson/hep/model.py

```diff
@@ -26,4 +26,19 @@
 from dojson import utils
 
 hep = Overdo()
+
+
+@hep.over('book', '^773..')
```

Comment:
@jmartinm  you are right, thats what I wanted to discuss also. This thing that I did is not running for every record. Sometimes it is missing.